D/DuckDB/build_tarballs.jl

```diff
@@ -10,32 +10,40 @@ sources = [
     GitSource("https://github.com/duckdb/duckdb.git", "f680b7d08f56183391b581077d4baf589e1cc8bd"),
 ]
 
-# Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/duckdb/
 
-cmake -B build \
-      -DCMAKE_INSTALL_PREFIX=${prefix} \
-      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
-      -DCMAKE_BUILD_TYPE=Release \
-      -DENABLE_SANITIZER=FALSE \
-      -DBUILD_EXTENSIONS='autocomplete;icu;parquet;json;fts;tpcds;tpch' \
-      -DENABLE_EXTENSION_AUTOLOADING=1 \
-      -DENABLE_EXTENSION_AUTOINSTALL=1 \
-      -DBUILD_UNITTESTS=FALSE \
-      -DBUILD_SHELL=TRUE \
-      -DDUCKDB_EXPLICIT_PLATFORM="${target}"
-cmake --build build --parallel ${nproc}
-cmake --install build
-
-if [[ "${target}" == *-mingw32 ]]; then
-    install -Dvm 755 "build/src/libduckdb.${dlext}" -t "${libdir}"
+if [[ "$target" == *-ming* ]]; then
+  build_command="cmake -B build -G \"Ninja\""
```

Comment:
Why is the build command different at all across platforms?  Either of the two should be just fine everywhere.

Reply:
Just because the official build instructions for windows use `-G "Ninja`. The library failed to precompile on windows, so I was just guessing that if I use the windows-specific instructions more closely, then that might fix the issue.

As you can see by the commit-history, I am just spitballing, and have no idea what I am doing. Help appreciated.